### ISSUE TITLE
New-DscResourcePowerShellHelp and Composite resources does not work

### ISSUE THREAD

**Issue by ChristophHannappel** [dsccommunity/DscResource.DocGenerator#115]
#### Details of the scenario you tried and the problem that is occurring
Trying to use the command `New-DscResourcePowerShellHelp` with a composite resource.
Composite sits in `source\DSCResources\CompositeName\` with two files
* CompositeName.psd1
* CompositeName.Schema.psd1

After looking at the [function `New-DscResourcePowerShellHelp`](https://github.com/dsccommunity/DscResource.DocGenerator/blob/6e55c6f778cf2b7b3fb846bda443929dfca42972/source/Public/New-DscResourcePowerShellHelp.ps1) it seams there is no code for composites

#### Steps to reproduce the problem
PS> New-DscResourcePowerShellHelp -ModulePath .\source\ -Verbose

#### Expected behavior
It creates some files.
#### Current behavior
It creates nothing
#### Suggested solution to the issue
It creates PowerShell compatible help files or the help file is updated, that the command can only do mof and class based.
#### The operating system the target node is running
Windows 10
#### Version and build of PowerShell the target node is running
Name |                          Value|
----                           |-----|
PSVersion                      |5.1.19041.1320|
PSEdition                      |Desktop|
PSCompatibleVersions           |{1.0, 2.0, 3.0, 4.0...}|
BuildVersion                   |10.0.19041.1320|
CLRVersion                     |4.0.30319.42000|
WSManStackVersion              |3.0|
PSRemotingProtocolVersion      |2.3|
SerializationVersion           |1.1.0.1|

#### Version of the module that was used
Name                     |Version| Path
----                     |------- |----
DscResource.DocGenerator |0.10.3  | `C:\Users\<Username>\git Repositories\<ModuleName>\output\RequiredModules\DscResource.DocGenerator\0.10.3\DscResource.DocGenerator.psd1`

**Comment by johlju** [comment]
That is correct that the module is missing support for conceptual help for composites. Wiki documentation is generated for composites, but not conceptual help yet. Happy to review a PR that adds that.
